ios/RNDeviceInfo/RNDeviceInfo.m

```diff
@@ -103,10 +103,16 @@ - (NSString*) deviceName
                               @"iPhone9,4" :@"iPhone 7 Plus",   // (model A1784 | Global)
                               @"iPhone10,3":@"iPhone X",        // (model A1865, A1902)
                               @"iPhone10,6":@"iPhone X",        // (model A1901)
+                              @"iPhone11,2":@"iPhone XS",       //
```

Comment:
Ahh sorry @duro! I thought you would've seen these other duplicate keys too. One for `@"iPhone11,2"` (lines 106 and 112) and another for `@"iPhone11,4"` (lines 107 and 113). Those are all of them, promise! 🤦‍♂️

Reply:
Cleaned those out